wsf_scraping/spiders/unicef_spider.py

```diff
@@ -0,0 +1,119 @@
+import scrapy
+from urllib.parse import urlparse
+from scrapy.http import Request
+from wsf_scraping.items import UNICEFArticle
+from scrapy.spidermiddlewares.httperror import HttpError
+from twisted.internet.error import DNSLookupError
+from twisted.internet.error import TimeoutError
+
+
+class UnicefSpider(scrapy.Spider):
+    name = 'unicef'
+
+    custom_settings = {
+        'JOBDIR': 'crawls/unicef'
+    }
+
+    def on_error(self, failure):
+        self.logger.error(repr(failure))
+
+        if failure.check(HttpError):
+            response = failure.value.response
+            self.logger.error(
+                'HttpError on %s (%s)',
+                response.url,
+                response.status,
+            )
+
+        elif failure.check(DNSLookupError):
+            request = failure.request
+            self.logger.error('DNSLookupError on %s', request.url)
+
+        elif failure.check(TimeoutError):
+            request = failure.request
+            self.logger.error('TimeoutError on %s', request.url)
```

Comment:
[Rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)): this method appears in two other spiders. Consider putting it in a base class, and having your spiders inherit from that?

Reply:
Sounds like a good idea 👍

Reply:
👍 i like the rule of three :)